slack_channel_report.py

```diff
@@ -89,23 +90,16 @@ def make_header(self, ur, pur, cid):
         blocks.append(self.membercount(cid, ur['start_date'], ur['end_date']))
         blocks.append(self.messages(cid, ur, pur))
         blocks.append(self.sf.divider())
-        blocks += self.users(cid, ur, pur)
-        #m = m.format(self.sf.comparison(us, pus, ['count', 1]), self.sf.comparison(us, pus, ['count', 0]))
-        #m += "\n"
-        #m += "That made you the *{}*-ranked poster on the Slack and meant you contributed "
-        #m += "*{:.1f}%*{} of this Slack's total public volume"
-        #m = m.format(utils.rank(us['rank']), us['percent_of_words'], self.sf.comparison(us, pus, ['percent_of_words'], False))
-        #tm = us.get("thread_messages")
-        #if tm:
-            #t = "In total, {} messages were posted as threaded responses to your messages.\n"
-            #t = t.format(self.sf.comparison(us, pus, ['thread_messages']))
-            #m += t
-        #blocks.append(self.sf.text_block(m))
         return blocks
 
     def make_report(self, ur, pur, cid):
+        """
+        if abbreviated, just send the header
```

Comment:
What does this comment mean?

Reply:
This was vestigial and no longer relevant.  PR amended to remove this string.